default/stringtables/en.txt

```diff
@@ -10039,12 +10039,42 @@ DESC_ORDERED_BOMBARDED
 DESC_ORDERED_BOMBARDED_NOT
 ''' that is not being bombarded by an object %1%'''
 
+DESC_AND_BEFORE_OPERANDS
+''''''
```

Comment:
I think it would be most clear, and most consistent between negated and not negated, to render ```A And B``` as ```(A and B)```

so this entry would be 
```'''('''```